src/components/CodeAssignmentModal/index.jsx

```diff
@@ -145,7 +146,10 @@ class CodeAssignmentModal extends React.Component {
 
       errors[csvFileKey] = message;
       errors._error.push(message);
-    } else if (csvEmails && numberOfSelectedCodes && csvEmails.length > numberOfSelectedCodes) {
+    } else if (
+      csvEmails && numberOfSelectedCodes && couponType !== ONCE_PER_CUSTOMER &&
```

Comment:
i think we should include MULTI_USE codes as well, because they can be assigned to multiple users too.

Reply:
@brittneyexline This now includes MULTI_USE codes as well!